src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -451,11 +453,29 @@ public void salvaCodiceIUN(){
         setIun(IUN);
     }
 
+    @And("Salva Api key")
+    public void salvaApiKey(){
+        String apiKey = dettaglioNotificaMittenteSection.getApiKey();
+        setApiKey(apiKey);
+    }
+
+    @And("Si verifica che Api Key sono diversi")
+    public void verificaDiversiApiKey(){
+        String actualApiKey = dettaglioNotificaMittenteSection.getApiKey();
+        Assert.assertFalse(actualApiKey.equalsIgnoreCase(ApiKey));
+        logger.info("Api Keys sono diversi");
+    }
+
     @And("viene inserito codice IUN salvato")
     public void vieneInseritoIunSalvato() {
         dettaglioNotificaMittenteSection.insertIunSalvatoAndRicercaOnPage(Iun);
     }
 
+    @And("ricerca notifica con IUN salvato")
```

Comment:
this never used

Reply:
this method is used for other scenarios, which are in blocked waiting for bug to be fixed